Libraries/LibJS/Tests/String.prototype.trim.js

```diff
@@ -0,0 +1,54 @@
+try {
```

Comment:
This needs

```suggestion
load("test-common.js");

try {
```

since https://github.com/SerenityOS/serenity/pull/1787 was merged